.github/workflows/publish-environment.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish execution environment
+
+on:
+  workflow_call:
+    inputs:
+      destination_dir:
+        required: true
+        type: string
+
+jobs:
+  publish-site:
+    name: Publish execution environment
+    runs-on: ubuntu-latest
+    environment: deploy-prod
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Ensure `destination_dir` is not empty
+        if: ${{ inputs.destination_dir == '' }}
+        run: exit 1
+      - uses: actions/cache@v3
+        id: restore-build
```

Comment:
Should we build the execution env with LM here instead of restoring?

Reply:
I don't think it's necessary. This is the build from the `publish-release` workflow, which is also published to NPM.